src/test/java/com/wikia/webdriver/testcases/templateclassification/TemplateClassificationTest.java

```diff
@@ -1,44 +1,55 @@
 package com.wikia.webdriver.testcases.templateclassification;
 
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.elements.oasis.components.templateclassificiation.TemplateClassification;
 import org.testng.annotations.Test;
 import com.wikia.webdriver.common.core.annotations.Execute;
 import com.wikia.webdriver.common.core.annotations.InBrowser;
 import com.wikia.webdriver.common.core.drivers.Browser;
 import com.wikia.webdriver.common.core.helpers.User;
 import com.wikia.webdriver.common.templates.NewTestTemplate;
-import com.wikia.webdriver.elements.oasis.pages.TemplateEditPage;
 import com.wikia.webdriver.elements.oasis.pages.TemplatePage;
 
 @Execute(asUser = User.STAFF, onWikia = "aga")
 @InBrowser(browser = Browser.CHROME)
 @Test(groups = {"templateClassification"})
 public class TemplateClassificationTest extends NewTestTemplate {
 
-  @Test(groups = "templateClassification_openAndClose")
-  public void templateClassification_openAndClose() {
-    new TemplatePage()
-        .open("T")
-        .getTemplateClassification()
-        .open()
-        .close();
-  }
+  private String templateName = "T";
 
-  @Test(groups = "templateClassification_changeTemplateType")
-  public void templateClassification_changeTemplateType() {
-    new TemplatePage()
-        .open("T")
-        .getTemplateClassification()
-        .open()
-        .changeTemplateType()
-        .close();
-  }
+  @Test(groups = "templateClassification_createTemplateAndChangeItsType")
+  public void templateClassification_createTemplateAndChangeItsType() {
+    TemplatePage templatePage = new TemplatePage();
+
+    TemplateClassification templateClassification = templatePage
+            .createTemplate(this.templateName)
+            .open(this.templateName)
+            .getTemplateClassification();
+
+    templateClassification
+            .resetTemplateType();
+
+    Assertion.assertTrue(templateName.equals("Unknown"), "Template type was reset");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "Unknown" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1132)